packages/messageport/tizen/src/messageport_manager.cc

```diff
@@ -0,0 +1,244 @@
+// Copyright 2021 Samsung Electronics Co., Ltd. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "messageport_manager.h"
```

Comment:
You could have just renamed the class to `MessagePort` without renaming the file, although it doesn't really matter.